docs/RELEASE_POLICY.md

```diff
@@ -0,0 +1,93 @@
+# Firecracker Release Policy
+
+This document describes Firecracker release planning, API support and the
+Firecracker release lifetime. Firecracker provides this Release Policy to help
+customers effectively plan their Firecracker based operations.
+
+## Firecracker releases
+
+Firecracker uses [semantic versioning](http://semver.org/) for all releases.
+Semantic versions are comprised of three fields in the form:
+
+    vMAJOR.MINOR.PATCH
+
+For example: v0.20.0, v0.22.0-beta5, and v99.123.77+foo.bar.baz.5.
+
+Firecracker publishes major, minor and patch releases:
+* Patch release - When the `PATCH` field is increased, the new release resolves
+  bugs and/or security issues without changing existing behavior or
+  user interface. Upgrade is recommended. Patch releases are created every time
+  when a security issue arises in a supported release.
+* Minor release - When the `MINOR` field is increased, the new release adds
+  new features, bug fixes or both without changing existing user interface or
+  user facing functionality. Adding new APIs can be done in a `MINOR`
+  Firecracker release as long as it doesn’t change the functionality of the APIs
+  available in the previous release. Minor releases are shipped when features
+  are ready for production. Firecracker team may bundle multiple features
+  under the same release.
+* Major release -  When the `MAJOR` field is increased, the new release adds
+  new features, bug fixes or both, changing existing user interface or user
+  facing functionality possibly that may make it incompatible with previous
```

Comment:
Maybe we can split this phrase to make it easier to read. Something like `When the MAJOR field is incremented, the release adds new features and/or bug fixes, changing the existing user interface or user facing functionality. This may make the release incompatible with previous ones.`.